app/src/main/java/com/dessalines/rankmyfavs/ui/components/favlist/TierListScreen.kt

```diff
@@ -246,89 +401,239 @@ fun TierSection(
                 )
             }
         }
+
+        if (editTierList) {
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.SpaceEvenly,
+            ) {
+                IconButton(onClick = {
+                    tierListViewModel?.swapTierOrders(tier, -1)
+                }) {
+                    Icon(
+                        imageVector = Icons.Filled.ArrowUpward,
+                        contentDescription = stringResource(R.string.move_up),
+                    )
+                }
+                IconButton(onClick = {
+                    tierListViewModel?.swapTierOrders(tier, 1)
+                }) {
+                    Icon(
+                        imageVector = Icons.Filled.ArrowDownward,
+                        contentDescription = stringResource(R.string.move_down),
+                    )
+                }
+
+                Spacer(modifier = Modifier.height(SMALL_PADDING))
+
+                FloatingActionButton(
+                    onClick = {
+                        tierListViewModel?.delete(tier)
+                    },
+                    shape = CircleShape,
+                    modifier =
+                        Modifier
+                            .imePadding()
+                            .size(28.dp),
```

Comment:
Move all these sizes to the same place that the `SMALL_PADDING` and such live, and name them appropriately.